mlflow/store/model_registry/abstract_store.py

```diff
@@ -69,9 +69,12 @@ def delete_registered_model(self, name):
         pass
 
     @abstractmethod
-    def list_registered_models(self):
+    def list_registered_models(self, page_token, max_results):
```

Comment:
nit: `(self, max_results, page_token)` seems more intuitive to me and it matches the ordering in the proto. we might as well make it a convention to follow the ordering in the proto.

Reply:
Updated the function signature!